e2e/runner/solana.go

```diff
@@ -813,3 +815,121 @@ func (r *E2ERunner) WaitAndVerifySPLBalanceChange(
 		return
 	}
 }
+
+// SetupTestALTWithRandomWallets sets up ALT with random accounts provided in setup solana script, with accounts provided as argument
+// used to test ALTs with large amount of accounts
+func (r *E2ERunner) SetupTestALTWithRandomWallets(accounts []solana.PublicKey) (solana.PublicKey, []solana.PublicKey) {
+	privkey := r.GetSolanaPrivKey()
+
+	recentSlot, err := r.SolanaClient.GetSlot(r.Ctx, rpc.CommitmentFinalized)
+	require.NoError(r, err)
+
+	// prefunded random wallets used for testing big number of accounts in payload
+	randomWallets := []solana.PublicKey{
+		solana.MustPublicKeyFromBase58("4C2kkMnqXMfPJ8PPK5v6TCg42k5z16f2kzqVocXoSDcq"),
```

Comment:
The maximum addresses accepted by a non-ALT transaction is `32`, and this is because Solana enforces a `1232 bytes` cap on its transaction size. To verify that ALT works in e2e test and make live network test easier, could we use fewer addresses for test, e.g. `33` or `34` or `35`? And, if we we devOps team to send funds to these static addresses, we could declare address list in a `var` block so they see it clearly.

Reply:
my idea was that we will create ALT out of e2e tests for live networks and just provide ALT for test in argument so we dont have to create new ALT for every run

that would require script to create and extend ALT, wdyt is better approach?